src/vtab/mod.rs

```diff
@@ -165,11 +165,28 @@ pub fn eponymous_only_module<'vtab, T: VTab<'vtab>>() -> &'static Module<'vtab,
     }
 }
 
+/// Virtual table configuration options
+#[repr(i32)]
+#[non_exhaustive]
+#[cfg(feature = "modern_sqlite")] // 3.7.7
+pub enum VTabConfig {
```

Comment:
Any reason not to derive at least `#[derive(Debug, Clone, Copy, PartialEq)]`?

Reply:
I will